docs/source/tutorial/modules.rst

```diff
@@ -74,14 +74,15 @@ The ``with`` keyword in expressions comes in two variants:
 This is particularly useful with ``do`` notation, where it can often improve
 error messages: ``with MyModule.(>>=) do ...``
 
-There is no formal link between the module name and its filename,
-although it is generally advisable to use the same name for each. An
-``import`` statement refers to a filename, using dots to separate
-directories. For example, ``import foo.bar`` would import the file
-``foo/bar.idr``, which would conventionally have the module declaration
-``module foo.bar``. The only requirement for module names is that the
-main module, with the ``main`` function, must be called
-``Main`` — although its filename need not be ``Main.idr``.
+If a file contains a module declaration ``module Foo.Bar.MyModule``, its
+path relative to the ``.ipkg`` project file (or the directory you are running
```

Comment:
This path is relative not to the `ipkg` project file, but source directory mentioned in this file. Say, it's written in the `ipkg` that `sourcedir = "src"` -- this means that path must be `src/Foo/Bar/MyModule.idr`. But by default `sourcedir` is indeed `.`, i.e. the directory where the `ipkg` file resides.

Reply:
You are right! Misread your comment at first, sorry. is this better?

Reply:
> is this better?

It is